projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -20,7 +20,11 @@
     # 'linalg.depthwise_conv_2d_nchw_chw' op inferred input/output operand #1 has shape's dimension #0 to be 4, but found 8
     "Conv2dWithPaddingDilationStrideStaticModule_depthwise_multiplier",
     "IscloseStaticModule_basic",
-    "IscloseStaticModuleTrue_basic"
+    "IscloseStaticModuleTrue_basic",
+    "AdaptiveMaxPool2dNonUnitOutputSizeStaticModule_basic",
+    "AdaptiveMaxPool2dNonUnitOutputSizeDynamicModule_basic",
+    "AdaptiveMaxPool2dUnitOutputSizeStaticModule_basic",
+    "AdaptiveMaxPool2dUnitOutputSizeDynamicModule_basic"
```

Comment:
Is the plan for this patch to also add 2d support? If not, it makes more sense to me to wait until 2d support is added to add these tests. It makes it easier when reviewing patches to see what cases are being tested.

Reply:
Perhaps I should make this a draft pr? I did actually intend to implement all the adaptive ops right away, as long as people didn't think the general idea was too gross. 

If I can make some helper functions to do a lot of the boilerplate stuff in this conversion, it would be very easy to write conversions for all of the other adaptive ops. 

What do you think would be better? Just getting this pull request through and then making a new pr for the other adaptive ops, or put all of them in one pr?

Reply:
In general smaller patches are better. But as ramiro suggested, I'd keep the tests with the commit that adds the code it tests.